includes/public/layouts/style1.php

```diff
@@ -75,7 +75,14 @@
 						</div>
 						<div class="wppr-review-grade-option-rating wppr-default">
 							<span class="<?php echo $review_object->get_rating_class( $review_option_rating ); ?>"
-								  style="width:<?php echo esc_attr( $review_option_rating ); ?>%;"></span>
+							<?php
+							if ( function_exists( 'ampforwp_is_amp_endpoint' ) && ampforwp_is_amp_endpoint() || function_exists( 'is_amp_endpoint' ) && is_amp_endpoint() ) {
+								echo 'style="min-width:'; // AMP style sanitizer coverts inline style's 'width' to 'max-width' which won't work.
+							} else {
+									echo 'style="width:';
```

Comment:
cant we add both min-width and width instead of doing that if/else statement. You can check if will be any difference

Reply:
Yes. Actually what AMP sanitizer does is, when any inline style property 'width' comes, it replaces it by 'max-width' ['here is the code'](https://github.com/ahmedkaludi/accelerated-mobile-pages/blob/b4a9d96b75d992bb932b5d769341286382013a55/includes/vendor/amp/includes/sanitizers/class-amp-style-sanitizer.php#L95) 

And the rating bars in our style2, don't work with max-width property.

Reply:
Sure, it'll work with amp the page. But, with the non-AMP page, It'll break and the bars won't show. If anyhow, we only use 'min-width' property, the animations of the bars won't show up.

Reply:
I thinking to use both, with and min-width, would this fix both cases? i mean to have also witdth and min-width rules inside.

Reply:
Got it. Made the changes.

Reply:
It will show the rating bars. But if in non-AMP version, the animations of filling bars won't be shown. If we use style="min-width: 50%; width: 50%;" or style="width: 50%; min-width: 50%;"

Reply:
Applied the filter and made changes.